spec/features/pooling_multiple_plates_spec.rb

```diff
@@ -67,25 +44,42 @@
             {
               'source_uuid' => plate_uuid,
               'source_location' => 'A1',
-              'destination_uuid' => child_plate_uuid,
+              'destination_uuid' => child_plate.uuid,
               'destination_location' => 'A1'
             },
             {
               'source_uuid' => plate_uuid_2,
               'source_location' => 'A1',
-              'destination_uuid' => child_plate_uuid,
+              'destination_uuid' => child_plate.uuid,
               'destination_location' => 'B1'
             },
             {
               'source_uuid' => plate_uuid_2,
               'source_location' => 'B1',
-              'destination_uuid' => child_plate_uuid,
+              'destination_uuid' => child_plate.uuid,
               'destination_location' => 'B1'
             }
           ]
         }
       },
-      body: json(:plate_creation, child_uuid: child_plate_uuid)
+      body: json(:plate_creation, child_uuid: child_plate.uuid)
+    )
+  end
+
+  let(:child_plate) { create :v2_plate, purpose_uuid: 'child-purpose-0', purpose_name: 'Pool Plate', barcode_number: 3 }
+
+  let(:pooled_plate_creation) do
+    response = double
+    allow(response).to receive(:child).and_return(child_plate)
+
+    response
+  end
+
+  def expect_pooled_plate_creation
+    expect_api_v2_posts(
+      'PooledPlateCreation',
+      [{ child_purpose_uuid: 'child-purpose-0', parent_uuids: [plate_uuid, plate_uuid_2], user_uuid: user_uuid }],
```

Comment:
can we use child_plate.purpose_uuid here instead of the string? Same applies to background block below